### PR TITLE
Add wavenum plot option

### PR DESCRIPTION
**Pull request by karllark**
Adding `wavenum` option to plotting of extinction curves.  Plotting versus 1/lambda is standard for UV extinction curve work - so useful.

Also have fixed a couple of bugs in the AverageExtData function.

### CODE REVIEW

**Comment by karllark** [comment]
@mdecleir : review possible?

**Comment by mdecleir** [comment]
> @mdecleir : review possible?

Possible, but probably not today.

File: measure_extinction/extdata.py [hunk 1]
```diff
@@ -1113,21 +1118,18 @@ def plot(
             if curtype in exclude:
                 continue
             # replace extinction values by NaNs for wavelength regions that need to be excluded from the plot
-            self.exts[curtype][self.npts[curtype] == 0] = np.nan
+            bdata = self.npts[curtype] == 0
+            if np.any(bdata):
```
Comment (mdecleir) [inline measure_extinction/extdata.py:1122]: Not sure why this is needed

**Review by mdecleir — COMMENTED** [review]
Looks good, just 1 question below.

File: measure_extinction/extdata.py (hunk 1 above)
Comment (karllark) [reply measure_extinction/extdata.py:1122]: I was having an issue at one point, but maybe that was a red herring.  I will revert to the more compact (and pythonic?) way.

**Comment by karllark** [comment]
Merging as failing tests are known other issues.

**Pull request merged**